bedrock/mozorg/templates/mozorg/about-2019.html

```diff
@@ -37,7 +37,7 @@ <h1 class="mzp-c-hero-title">{{ _('Mozilla makes browsers, apps, code and tools
       </div>
 
       <p class="mzp-c-hero-cta">
-        <a id="read-mission-button" class="mzp-c-button" href="{{ url('mozorg.mission') }}" data-button-name="Read Our Mission" data-link-type="button" data-cta-position="primary cta">{{ _('Read Our Mission') }}</a>
+        <a id="read-mission-button" class="mzp-c-button" href="{{ url('mozorg.mission') }}" data-button-name="Read Our Mission" data-cta-position="primary cta">{{ _('Read Our Mission') }}</a>
```

Comment:
If these are links then should we be using `data-link-type="button"` and `data-link-name="Read Our Mission"` instead?

I'm actually not 100% sure on this, so I asked the question in the issue for clarification:

https://github.com/mozilla/bedrock/issues/7392#issuecomment-509545915